WORKSPACE

```diff
@@ -165,6 +159,70 @@ http_archive(
     ],
 )
 
+load("@google_mediapipe//third_party:external_files.bzl", "external_files")
+external_files()
+
+new_local_repository(
+    name = "linux_openvino",
+    build_file_content = """
+load("@bazel_skylib//lib:paths.bzl", "paths")
+PREFIX = "runtime"
+cc_library(
+    name = "openvino",
+    srcs = glob(
+        [
+            paths.join(PREFIX, "lib/intel64/libopenvino_c.so"),
+            paths.join(PREFIX, "lib/intel64/libopenvino.so"),
+        ],
+    ),
+    hdrs = glob(
+        [
+            paths.join(PREFIX, "include/ie/**/*.h"),
+            paths.join(PREFIX, "include/**/*.hpp"),
```

Comment:
I think just one header should suffice? Does any of the OV tensor converter calculators use different header than openvino.hpp?